packages/protocol/scripts/bash/ganache.sh

```diff
@@ -9,5 +9,5 @@ yarn run ganache-cli \
   --gasPrice 0 \
   --networkId 1101 \
   --gasLimit 10000000 \
-  --defaultBalanceEther 1000000 \
+  --defaultBalanceEther 200000000 \
```

Comment:
because the inital ether balance has been increased in all other ganache servers (required by the new migtrations config), so I thought that if one was to run it on its own and deploy himself, it would be nice to have the same ether balance s in the other scripts